R/data-model-helpers.R

```diff
@@ -245,3 +245,50 @@ data_model_db_types_to_R_types <- function(data_model) {
   data_model$columns$type <- new_type
   data_model
 }
+
+upd_pks_after_rename <- function(
+  pks,
+  table_name,
+  list_of_renames) {
+
+  pk_of_table <- filter(pks, table == !!table_name) %>% pull(column)
```

Comment:
We could also express with `left_join()` and `coalesce()` : 

```r
rename_logic <- ...
new_pks <-
  pks %>%
  left_join(rename_logic) %>%
  mutate(table = coalesce(new_table, table))
```

No need to spend time here, though.